ui/src/middleware.ts

```diff
@@ -1 +1,5 @@
 export { default } from "next-auth/middleware";
+
+export const config = {
+  matcher: ["/((?!_next/static|_next/image|favicon.ico|signin|onboarding).*)"],
```

Comment:
These pages are reachable without a session turned on. Where do we configure the "redirectUrl" when they navigate to on of those pages while a session exists.

For example:

(Logged in) Navigate to /signin -> Redirected to /dashboard
(Logged in) Navigate to /onboard -> Redirected to /dashboard
(Logged out) Navigate to /dashboard -> Redirected to /signin

Reply:
At the moment this happens on a per page basis since there are only two public pages that shouldn't be accessible without a session. 

Like this:
  const session = await getServerSession();
  if (session) {
    redirect("/dashboard");
  }

Could add this to the middleware function instead if you want to.